api/src/main/java/org/eclipse/microprofile/openapi/models/Paths.java

```diff
@@ -56,27 +48,38 @@
     void removePathItem(String name);
 
     /**
-     * Returns an immutable map of the path items.
+     * Returns a copy map (potentially immutable) of the path items.
```

Comment:
minor thing, but in some of the other methods we have default implementations that rely on `getPathItems` to never return null.   Perhaps we should clarify in this javadoc that you must always return a non-null map, even if empty?